llvm/test/CodeGen/AArch64/complex-deinterleaving-symmetric-fixed.ll

```diff
@@ -0,0 +1,76 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
```

Comment:
I suppose this patch doesn't affect fixed-width? This test shows the same output before and after patch

Reply:
Yeah so the pass already handles `@simple_symmetric_muladd2`, but I didn't add support for the fixed-width patterns in `@simple_symmetric_muladd4` mainly because I'd expect the VectorCombine pass to have already removed the shuffles before we reach the complex deinterleaving pass. However, if we find cases where actually it would be useful to support fixed-width vectors it should be easy enough to add.